data/core/faqs.md

```diff
@@ -807,39 +774,10 @@
     - utter_stack_details
     - utter_explain_nlucore
 * deny
-    - utter_tryout
-* ask_faq_platform OR ask_faq_languages OR ask_faq_tutorialcore OR ask_faq_tutorialnlu OR ask_faq_opensource OR ask_faq_voice OR ask_faq_slots OR ask_faq_channels OR ask_faq_differencecorenlu OR ask_faq_python_version OR ask_faq_community_size OR ask_faq_what_is_forum
-    - action_faqs
-    - utter_tryout
-* how_to_get_started{"product":"core"} OR affirm OR how_to_get_started{"product":"stack"}
-    - utter_quickstart
-
-
-## new to rasa/bots, explain and skip to installation
-* greet
-    - action_greet_user
-* how_to_get_started
-    - utter_getstarted
-    - utter_first_bot_with_rasa
-* affirm
-    - action_set_onboarding
-    - slot{"onboarding": true}
-    - utter_built_bot_before
-* deny
-    - utter_explain_stack
-    - utter_stack_details
-    - utter_explain_nlucore
-* affirm OR how_to_get_started{"product":"stack"}
-    - utter_explain_nlu
-    - utter_explain_core
-    - utter_tryout
+    - utter_direct_to_step2
 * ask_faq_platform OR ask_faq_languages OR ask_faq_tutorialcore OR ask_faq_tutorialnlu OR ask_faq_opensource OR ask_faq_voice OR ask_faq_slots OR ask_faq_channels OR ask_faq_differencecorenlu OR ask_faq_python_version OR ask_faq_community_size OR ask_faq_what_is_forum
     - action_faqs
-    - utter_tryout
-* deny
```

Comment:
i see we've gotten rid of the `deny` everywhere -- what if a user does say "no"?

Reply:
well we don't need to, i just thought it was weird we were getting rid of these haha

Reply:
So should I re-add it or not? :-D

Reply:
let's leave it for now